setup.py

```diff
@@ -1,24 +1,7 @@
-import os
-import sys
-
 from setuptools import find_packages
 from setuptools import setup
 
 
-# setup logic from github.com/jupyterhub/jupyterhub
-v = sys.version_info
-if v[:2] < (3, 6):
-    error = "ERROR: LTIAuthenticator requires Python version 3.6 or above."
-    print(error, file=sys.stderr)
-    sys.exit(1)
-
-# Get the current package version.
-here = os.path.abspath(os.path.dirname(__file__))
-version_ns = {}
-with open(os.path.join("_version.py")) as f:
-    exec(f.read(), {}, version_ns)
-
-
```

Comment:
This section isn't consumed in setup() below by setting the version dynamically.